### PR TITLE
update byod.md

### PR DESCRIPTION
**Pull request by EThom73**
**What:**  
The BYOD Policy is finally out of Google Docs and in GitHub.
**Why**
The BYOD Policy is an important part of our IMS system.
**Notes:**  
The links are to documents that don't exist/exist only in draft. They will be updated once the structure of the IMS file is established

### CODE REVIEW

File: ims/company/byod.md [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+
+### Use endpoint protection software
+Unless your device is operating iOS or ChromeOS, your device must have endpoint protection software installed, which must also be kept up to date. 
+
+Endpoint protection is more than just antivirus software, and usually includes a firewall and may have other tools to protect your device, like network threat detection.
+
+Any device you plug into the device, like an external USB device, must also be scanned for viruses. We recommend doing a full scan at least once a month.
+
+### Set a password/passphrase
+Your device must require a password/passphrase to login, return from screensaver or wake from sleep. Ideally your device will lock the screen (or start the screensaver) after a few minutes (no more than 5) of the device being idle
```
Comment (lukemorton) [inline ims/company/byod.md]: What's the difference between a password and a passphrase?

File: ims/company/byod.md [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+
+### How you store & manage data
+If data is stored and managed incorrectly it can be lost; stolen or changed (including someone loading a virus). ‘Storing data’ includes on your computer (whether it is on or off) and portable media, like USB sticks. Data should be encrypted wherever possible.
+
+If you don’t need files and data any more, delete them and make sure they can’t be recovered.
+
+### Disk Encryption
+You must encrypt all your data storage devices. This includes:
+
+hard disks (including on smartphones and tablets)
```
Comment (lukemorton) [inline ims/company/byod.md]: Should this list be bulleted?

File: ims/company/byod.md (hunk 1 above)
Comment (EThom73) [reply ims/company/byod.md]: https://www.passworddragon.com/password-vs-passphrase

File: ims/company/byod.md (hunk 2 above)
Comment (EThom73) [reply ims/company/byod.md]: Good spot - cheers.

**Pull request merged**